manifests/vehicle-position-splitter/kuopio/overlays/dev/ksops-secret-generator.yaml

```diff
@@ -11,4 +11,4 @@ metadata:
           # otherwise, path should be relative to manifest files, like
           # path: ../../../ksops
 files:
-  - ./secret.yaml
\ No newline at end of file
+  - ./kuopio-secret.yaml
```

Comment:
Onko tuossa joku erillinen syy nimetä secret filu kaupungin mukaan?

Reply:
The reason behind is that we have multiple cities -> each city has its secret named after it